app/install/ubuntu.md

```diff
@@ -74,6 +74,15 @@ section on the page below, setting  *distribution* to the appropriate value ( ls
     should be performing migrations at a time. This limitation is lifted for
     Kong 0.15, 1.0, and above.
 
+    If you are going to run Kong in [DB-less mode](/{{site.data.kong_latest.release}}/db-less-and-declarative-config/),
```

Comment:
This one is also missing the change in the title, it only says "Prepare your database" (but like all the other similar ones, my comment about the order the info is presented also applies)